ospatch/updates.go

```diff
@@ -0,0 +1,116 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package ospatch
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+const (
+	rpmQuery = "/usr/bin/rpmquery"
+)
+
+func exists(path string) (bool, error) {
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return false, nil
+		}
+		return false, err
+	}
+
+	return true, nil
+}
+
+func getBtime(stat string) (int, error) {
```

Comment:
Can you document what "btime" is? It would also be useful to document the whole approach, maybe with `rpmReboot` since that's the public function. For example, "On systems using rpm, the best way to check if a reboot is required is to look at...". We'll also want that wording in our API so users know when a reboot would be triggered.